jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -264,7 +264,7 @@ public int compare(ProcessDefinition o1, ProcessDefinition o2) {
 						return o1.getName().compareTo(o2.getName());
 					} else if ("ProcessVersion".equals(queryContext.getOrderBy())) {
 						return o1.getVersion().compareTo(o2.getVersion());
-					} else if ("Project".equals(queryContext.getOrderBy())) {
+					} else if ("DeploymentId".equals(queryContext.getOrderBy())) {
```

Comment:
I'm concerned this could be a breaking change, we could perhaps keep both strings (Project and DeploymentId) instead. @mswiderski wdyt?

Reply:
@cristianonicolai Well, it is not a fully breaking change in a way that it breaks your code, but yes, if somebody had used QueryContext's orderBy with "Project", it will no longer filter properly. I agree with keeping both.